docs/source/tutorials/GetMetadataNCBI.md

```diff
@@ -0,0 +1,120 @@
+# Get Metadata from NCBI
+Here is a solution to search and extract metadata that are associated to the
+sequences that are available in NCBI databases, including SRA. With this tool
+you can extract large metadata datasets, and obtain metadata in database/table
+that are nicely formatted.  
+
+You can eg. get all the metadata from sequences you eg. have downloaded with
+[ncbi-genome-download](https://github.com/kblin/ncbi-genome-download) that
+is installed in the `ncbidown` conda environment on SAGA.
+
+Assume you have downloaded sequences, and you want to retrieve the data for all
+the BioSamples that belong to the genus _Providencia_.
+
+You want to find all the metadata for all the isolates you have, and you want to be able
+to use the biosample ID you have to link the metadata you will download to the
+sequences you already have downloaded. Note that here I will actually download more
+metadata than what I need, but its easy to filter out afterwards those I do not need.
+
+The metadata that will be downloaded is the results that are returned after a
+query to an NCBI database. A query can eg. be a specific taxon: species, subspecies.
+But a query can be more than that.
+
+The tool we will be using is : [NCBImeta](https://github.com/ktmeaton/NCBImeta)
+
+Example local installation
+```
+conda create -n ncbimeta -c bioconda ncbimeta
+```
+
+It is already installed on SAGA, to use it:
+```
+conda activate ncbimeta
+```
+
+To use ncbimeta, you will need to use an API (application programming interface) key from NCBI.
+An API key is an authentication code that will serves to tell NCBI that you are the user
+making requests to download metadata via ncbimeta software. Using an NCBI API key
+will give you the possibility to send many more queries to NCBI (eg. to download more metadata)
+per second than you would be able to perform without it.
+
+To get an API key from NCBI :
+- login (or create an account)
+- account > account settings : there you can create an API key you will use - copy it and keep it safe
+
+We will need to create a configuration file (.yalm) for each metadata download
+we will ask the software to do. The configuration file is not a configuration file to
+run the software, but it is a configuration file that specifies the search/query
+we are asking the software to do: to retrieve the available metadata we are interested in,
+and then the specification of the specific of fields we want to download in a database/table.  
+
```

Comment:
I suggest you somehwere here show a screenshot of what this file should look like, not only link to it.